packages/components/credentials/OpenSearchUrl.credential.ts

```diff
@@ -10,12 +10,26 @@ class OpenSearchUrl implements INodeCredential {
     constructor() {
         this.label = 'OpenSearch'
         this.name = 'openSearchUrl'
-        this.version = 1.0
+        this.version = 2.0
         this.inputs = [
             {
                 label: 'OpenSearch Url',
                 name: 'openSearchUrl',
                 type: 'string'
+            },
+            {
+                label: 'User',
+                name: 'user',
+                type: 'string',
+                placeholder: '<OPENSEARCH_USERNAME>',
+                optional: true
+            },
+            {
```

Comment:
it is better to have this on `this.credential`, example [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/vectorstores/Pinecone/Pinecone.ts#L35)

Reply:
This is the credential code, not the node code. The node is already using the OpenSearchURL credential, this PR only adds the `user` and `password` fields to it.

But I'm not sure if I understood your suggestion correctly.

Reply:
oh i guess I was half-dreaming when reviewing the code haha!